### DISCUSSION TITLE
Mention in README that `.gitignore`d files will not be deployed?

### DISCUSSION THREAD

**Discussion by sgasse** [JamesIves/github-pages-deploy-action#1457] [Ideas]
Hi! I am a big fan of this action and have been using it for a while. However there is one thing which I ran into already twice, so I wonder if it is worth documenting it in the README:
- Some build tools auto-generate a `.gitignore` for build output to avoid that users accidentally include the build output in git commits.
- This however means that it will not be deployed with this action although it shows up in the recursively listed directory (`ls -laR`).

Would it make sense to mention that somewhere? Just removing the `.gitignore` is enough in the CI pipeline to solve it, but you have to know about it.

**Comment by JamesIves** [comment]
I like that idea! Do you want to make a PR for the addition?

**Comment by sgasse** [comment]
OK nice :slightly_smiling_face: I was not sure where exactly to put it and added it to the `folder` argument now. Do you think it fits there or rather between the required and optional configuration keys?

https://github.com/JamesIves/github-pages-deploy-action/pull/1459
